bundles/net.tourbook.chart/src/net/tourbook/chart/ChartComponentGraph.java

```diff
@@ -3473,16 +3446,14 @@ private void drawAsync_530_BarGraph(final GC gcGraph, final GraphDrawingData dra
              * make sure the bars do not overlap
              */
             if (serieLayout != ChartDataYSerie.BAR_LAYOUT_SINGLE_SERIE) {
-               if (devXPosNextBar > 0) {
-                  if (devXPos < devXPosNextBar) {
+               if (devXPosNextBar > 0 && devXPos < devXPosNextBar) {
 
-                     // bars do overlap
+                  // bars do overlap
 
```

Comment:
The comment `// bars do overlap` is only for the inner `if` of the old code and not for the joined `if`

Reply:
Would that be more accurate like this ?

![image](https://user-images.githubusercontent.com/25235665/110135703-0f44e980-7d8c-11eb-9b1d-0e09d86a7490.png)

Reply:
> Would that be more accurate like this ?

Yes

Reply:
Ok, I fixed it in this [PR](https://github.com/wolfgang-ch/mytourbook/pull/313)